package.nls.json

```diff
@@ -9,11 +9,11 @@
     "settings.importStrategy.description": "Defines where `pylint` is imported from. This setting may be ignored if `pylint.path` is set.",
     "settings.importStrategy.useBundled.description": "Always use the bundled version of `pylint`.",
     "settings.importStrategy.fromEnvironment.description": "Use `pylint` from environment, fallback to bundled version only if `pylint` not available in the environment.",
-    "settings.interpreter.description": "When set to a path to python executable, extension will use that to launch the server and any subprocess.",
+    "settings.interpreter.description": "When set to a path to Python executable, extension will use that to launch the server and any subprocess.",
     "settings.showNotifications.description": "Controls when notifications are shown by this extension.",
     "settings.showNotifications.off.description": "All notifications are turned off, any errors or warning are still available in the logs.",
     "settings.showNotifications.onError.description": "Notifications are shown only in the case of an error.",
     "settings.showNotifications.onWarning.description": "Notifications are shown for errors and warnings.",
     "settings.showNotifications.always.description": "Notifications are shown for anything that the server chooses to show.",
     "settings.includeStdLib.description": "Controls whether to filter out files from standard library from being linted."
-}
```

Comment:
Last line is missing

Reply:
Could you please guide me which line is missing? 
Because when I compared it with current file in the repository all the lines were present. 
Thank you

Reply:
There should be a newline after the last `}`. You can use Prettier extension to format this.